packages/sku/lib/openBrowser/index.js

```diff
@@ -2,32 +2,63 @@
 // https://github.com/facebook/create-react-app/blob/master/packages/react-dev-utils/openBrowser.js
```

Comment:
Worth updating this comment to point to the actual commit rather than just `master`.